bin/install-wp-tests.sh

```diff
@@ -14,6 +14,10 @@ WP_VERSION=${5-latest}
 WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib}
 WP_CORE_DIR=${WP_CORE_DIR-/tmp/wordpress/}
 
+WP_PLUGIN_DIR=${WP_PLUGIN_DIR-/tmp/wordpress/wp-content/plugins}
```

Comment:
Remove use a filter to manipulate the active plugins

Reply:
Handled.